### PR TITLE
Karim app models

### PR DESCRIPTION
**Pull request by karimrattani**
added member registration field in user registration form

### CODE REVIEW

**Comment by mtfuller** [comment]
Sounds great! Didn’t know you could do this. Give me a chance to try it and we’ll merge to master

**Comment by mtfuller** [comment]
I ran into a few registration issues, but I believe I got them cleaned up. @karimrattani Can you verify on your end? I think we can merge to master afterwards.

File: event_manager/models.py [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+    role = models.CharField(max_length=200)
+    major = models.CharField(max_length=200)
+    points = models.IntegerField(default=0)
+
+    def __str__(self):
+        return self.user.get_full_name()
+
+class Team(models.Model):
+    title = models.CharField(max_length=200)
+    lead = models.ForeignKey(Member, null=True, on_delete=models.SET_NULL)
```
Comment (karimrattani) [inline event_manager/models.py]: We need to change lead foreign key from Member to User, then we will need to make migrations again. For me, migrations were not changing anything, so I went into database manually and change member_id to user_id in the table event_manager_team_members

**Comment by karimrattani** [comment]
i updated the branch with new changes, i don't know if merging this would reflect those changes. I don't have any option to make pull request again on that branch. Should I close this request and make another one?

**Comment by mtfuller** [comment]
The pull request is still active. Let me try out the new changes, and if all goes well I'll merge. Sound good?

**Comment by karimrattani** [comment]
yup sounds good to me

**Comment by mtfuller** [comment]
Very nice, @karimrattani . I really like what you did, changing members fields to User. Works much better.

**Pull request merged**